include/ImmediateContext.inl

```diff
@@ -496,6 +496,12 @@ inline void ImmediateContext::PreDraw() noexcept(false)
     PIXScopedEvent(GetGraphicsCommandList(), 0ull, L"PreDraw");
     PreRender(COMMAND_LIST_TYPE::GRAPHICS);
 
+    if (m_bUseRingBufferDescriptorHeaps)
+    {
+        // Always dirty the state when using ring buffer heaps because we can't safely reuse tables in that case.

```

Comment:
In the original this was above PreRender

Reply:
Same as above.